.drone.star

```diff
@@ -8,7 +8,7 @@ def main(ctx):
       'ldap': 'https://marketplace.owncloud.com/api/v1/apps/user_ldap/0.15.0',
       'ldap_sha': '742703c66a8a112636c30aa500d7b4fdff670bac179fe3ae658a2b4016e5847f',
       'php': '7.3',
-      'behat_version': '10.4.0',
+      'behat_version': 'behat-10.4.0',
```

Comment:
branches do not work in this setup.

See at the bottom. It adds a `v` prefix

Reply:
Maybe skip all that logic?
It is not documented, was only a workaround

The next unlucky guy will struggle again.